about.php

```diff
@@ -2,19 +2,28 @@
 <!DOCTYPE html>
 <html lang="en">
 <?php require_once __DIR__ .'/template/head.phtml';?>
-<title><?php $titleAbout?></title>
+<?php
+function getPageTitle($page_titles) {

```

Comment:
@ReginaBorisova 
Can you declare the function once and use on every page, please?

Reply:
all done :)